identifiers/country-pa/README

```diff
@@ -0,0 +1,9 @@
+Open Civic Data Divisions: Panama
+
+File contents in `country-pa` directory:
+*   top_level.csv: contains country level OCD-ID
+*   provinces.csv: contains the list of provinces in Panama [https://en.wikipedia.org/wiki/Provinces_of_Panama]
+*   electoral_districts.csv: contains list of electoral districts (39 circuits) in Panama
```

Comment:
Is there an official source for these electoral districts that could be linked in the readme? It'd also help with checking the `electoral_districts.csv` file for correctness

Reply:
We were using mainly this page: https://www.iso.org/obp/ui/#iso:code:3166:PA

Additional ones:
https://en.wikipedia.org/wiki/ISO_3166-2:PA
https://alertas.directoriolegislativo.org/wp-content/uploads/2019/05/ENG-Post-election-report-Panama.pdf